### PR TITLE
feat(FEC-7808): quality watcher for the player

### PR DESCRIPTION
**Pull request by odedhutzler**
### Description of the Changes

Adding getCurrentQuality API for the quality watcher feature.
https://github.com/kaltura/playkit-js/pull/295

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/hls-adapter.js [hunk 1]
```diff
@@ -194,10 +200,23 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
...
+    this._capabilites.fpsControl = true;
     this._hls.subtitleDisplay = this._config.subtitleDisplay;
     this._addBindings();
   }
 
+  /**
+   * Get the current bandiwidth that is playing
+   * @returns {number} the current bandwidth
+   */
+  getCurrentQuality(): number {
```
Comment (OrenMe) [inline src/hls-adapter.js]: remove unused

Comment (odedhutzler) [reply src/hls-adapter.js]: fixed

File: src/hls-adapter.js [hunk 2]
```diff
@@ -136,6 +136,12 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
     if (Utils.Object.hasPropertyPath(config, 'playback.useNativeTextTrack')) {
       adapterConfig.subtitleDisplay = Utils.Object.getPropertyPath(config, 'playback.useNativeTextTrack');
     }
+    if (Utils.Object.hasPropertyPath(config, 'playback.recoverDecodingErrorDelay')) {
```
Comment (OrenMe) [inline src/hls-adapter.js]: remove this config, it doesn't exist on player or adapter

Comment (OrenMe) [reply src/hls-adapter.js]: and enable passing the fps settings:
fpsDroppedMonitoringPeriod and fpsDroppedMonitoringThreshold and enable flag capLevelOnFPSDrop.
All under a config called `abr`

Comment (odedhutzler) [reply src/hls-adapter.js]: fixed, changed according to a new config

File: src/hls-adapter.js [hunk 3]
```diff
@@ -136,6 +136,15 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
     if (Utils.Object.hasPropertyPath(config, 'playback.useNativeTextTrack')) {
       adapterConfig.subtitleDisplay = Utils.Object.getPropertyPath(config, 'playback.useNativeTextTrack');
     }
+    if (Utils.Object.hasPropertyPath(config, 'abr.fpsDroppedFramesInterval')) {
+      adapterConfig.fpsDroppedFramesInterval = config.playback.fpsDroppedFramesInterval;
```
Comment (OrenMe) [inline src/hls-adapter.js]: need to pass this and `fpsDroppedMonitoringThreshold`, capLevelOnFPSDrop via the hlsjs config, e.g. `adapterConfig.hlsConfig.xyz = ...`

Comment (OrenMe) [reply src/hls-adapter.js]: and move them to `config.abr`

**Pull request merged**